### ISSUE TITLE
Rebar unhappy [JIRA: RIAK-1572]

### ISSUE THREAD

**Issue by macintux** [basho/riak_ensemble#50]
Fresh checkout of `develop`. Doesn't seem to impact anything.

```
$ make
./rebar get-deps
WARN:  Expected /Users/John/github/Basho/riak_ensemble/deps/lager to be an app dir (containing ebin/*.app), but no .app found.
WARN:  Expected /Users/John/github/Basho/riak_ensemble/deps/eleveldb to be an app dir (containing ebin/*.app), but no .app found.
==> riak_ensemble (get-deps)
WARN:  Expected /Users/John/github/Basho/riak_ensemble/deps/lager to be an app dir (containing ebin/*.app), but no .app found.
WARN:  Expected /Users/John/github/Basho/riak_ensemble/deps/eleveldb to be an app dir (containing ebin/*.app), but no .app found.
```

**Comment by reiddraper** [comment]
Reproduced with `git clean -ffdx` too.

**Comment by cmeiklejohn** [comment]
Resolved by #60.
